### DISCUSSION TITLE
I can't get approved to the Student Developer Pack

### DISCUSSION THREAD

**Discussion by janasd** [community/community#54253] [GitHub Education]
### Hi everyone,

Hello,
I tried everything to get accepted to the Student Developer Pack, but every time it gets rejected. My webcam quality is good but because I have to cover a large area the text is not readable, so I would like to get ideas from the community, how you handled to get approved. I need to cover an English translation and the original student ID, but the signature of the school is on the back...

**Comment by samus-aran** [comment]
Hi @janasd 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

**Reply by janasd** [reply]
Hello, I got a school email, but I can't receive messages from emails with another domain, and it is another domain than the domain from the selection. The student ID is double page printed, so I can't show it in one picture.
